docs/.wordlist.txt

```diff
@@ -0,0 +1,49 @@
+addons
+API
+APIs
+balancer
+CharmHub
+CLI
+DEX
+dropdown
+Diátaxis
+EBS
+EKS
+favicon
+filesystem
+Grafana
+hostname
+IAM
+installable
+JSON
+Juju
+Kubernetes
+Kubeflow
+lifecycle
+Makefile
+MicroK
+MinIO
+MLflow
+MLOps
+MyST
+namespace
+namespaces
+NodePort
+observability
+OIDC
+OLM
+Permalink
+ReadMe
+reproducibility
+readthedocs
+reST
+reStructuredText
+RTD
+subdirectories
+subtree
+UI
+Jira
+MLflow
+mlflow
+tensorboard
```

Comment:
What happened to the alphabetical order here? Also MLflow is already included above and `mlflow` should most probably not be here? Then I see that they stylise the last one as [TensorBoard](https://www.tensorflow.org/tensorboard).

Reply:
Good catch, fixed